target/scripts/start-mailserver.sh

```diff
@@ -194,10 +194,16 @@ fi
 # marker to check if container was restarted
 date >/CONTAINER_START
 
+# Container logs will receive updates to this log file:
+MAIN_LOGFILE=/var/log/mail/mail.log
+# NOTE: rsyslogd would usually create this later during `_start_daemons`, however it would already exist if the container was restarted.
+touch "${MAIN_LOGFILE}"
+# Ensure `tail` follows the correct position of the log file for this container start (new logs begin once `_start_daemons` is called)
+TAIL_START=$(wc -l < "${MAIN_LOGFILE}")
```

Comment:
Increment by 1 via [arithimitic expansion](https://stackoverflow.com/questions/6348902/how-can-i-add-numbers-in-a-bash-script/6348945#6348945) bash syntax:

```suggestion
TAIL_START=$(( $(wc -l < "${MAIN_LOGFILE}") + 1 ))
```

Container restarts were otherwise displaying the last log line from the prior container run.